pyproject.toml

```diff
@@ -41,7 +41,7 @@ ecalc = 'ecalc_cli.main:main'
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
-pydantic = "^1.10"
+pydantic = "^2"
```

Comment:
Necessary to have "<3" or between 1 and 3?

Reply:
betyr ikke ^2 at den må være kompatibel med 2? dvs <3 implisitt?

Reply:
"^2" tar ikke med at vi støtter 1